From: Roland Stigge <stigge@antcom.de>
Date: Mon, 5 Sep 2016 16:23:07 +0100
Subject: manpage designs

Adjust the manpage by specifying further clock designs included in
the Debian distribution, specified by the -name option.
---
 swisswatch.man | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/swisswatch.man b/swisswatch.man
index 47201f6..c6ae985 100644
--- a/swisswatch.man
+++ b/swisswatch.man
@@ -205,11 +205,17 @@ If set, the clock will remain circular even if one tries to resize it
 to a non-circular ellipse.  This is the previous behavior.  The
 possible values are the same as for \fB.swiss/.Swiss\fP.
 .SH EXAMPLE RESOURCES
-The distributed application defaults files define three different
+The distributed application defaults files define various different
 looks for swisswatch.  Try them out by calling it as
 .sp 1
 swisswatch
 .sp 1
+swisswatch -name swisswatch
+.sp 1
+swisswatch -name fancy
+.sp 1
+swisswatch -name botta
+.sp 1
 swisswatch -name swissclock
 .sp 1
 swisswatch -name oclock
